packetbeat/route/route_test.go

```diff
@@ -0,0 +1,91 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
I think this file should have a build tag to match the OSes on which `Default()` is implemented.